code/modules/organs/subtypes/diona.dm

```diff
@@ -178,8 +178,22 @@
 	dislocated = -1
 	joint = "structural ligament"
 	amputation_point = "branch"
+	vital = FALSE // Lore team requested this, not vital organ. We can still live without it.
 
+/obj/item/organ/external/head/diona/New(loc, ...)
+	..()
+	if(owner)
+		owner.sdisabilities &= ~BLIND
+
+/obj/item/organ/external/head/diona/Destroy()
+	if(owner)
+		owner.sdisabilities |= BLIND
+	. = ..()
 
+/obj/item/organ/external/head/diona/removed(var/mob/living/user)
+	..()
+	if(owner)
+		owner.sdisabilities |= BLIND
```

Comment:
Shouldn't this be `user` or something?

Reply:
Also, why are you touching this at all. This should be handled by vision organ code.